frontend/src/app/shared/models/IModal.ts

```diff
@@ -0,0 +1,12 @@
+import { Type } from '@angular/core';
+
+export interface IModal {
+    header?: string,
+    content?: string,
+    cancelText?: string,
+    confirmText?: string
+    okText?: string
+    /* eslint-disable  @typescript-eslint/no-explicit-any */
+    component?: Type<any>
```

Comment:
I still don't like this any, but let it be then

Reply:
I don`t like it too, but in Angular library *ngComponentOutlet is taking type Type<any> and if we take any other - there will be an error
![image](https://user-images.githubusercontent.com/98846587/226328370-af36702d-dfb8-4654-a82d-e2c39dad7ed0.png)